libsplinter/src/store/mod.rs

```diff
@@ -91,6 +95,12 @@ pub fn create_store_factory(
                     "Failed to build connection pool".to_string(),
                 )
             })?;
+            let conn = pool
+                .get()
+                .map_err(|err| InternalError::from_source(Box::new(err)))?;
+            if !any_pending_postgres_migrations(&conn)? {
+                return Err(InternalError::with_message(String::from("The splinter database is out of date. You must run `splinter migrate` before running splinterd")));
```

Comment:
Please wrap these error messages at 100 characters

Reply:
the command is also `splinter database migrate`

Reply:
Cool! Made requested changes:
* Simplified `any_pending_postgres_migrations` logic to translate an error fetching the current version as a blank slate (I hadn't expected the API would behave that way, so it's good you caught it)
* Wrapped error messages to 100 characters
* Corrected cli command to say `splinter database migrate`